### DISCUSSION TITLE
Best practice for Lists (<ol> and <ul>)

### DISCUSSION THREAD

**Discussion by YousefED** [resend/react-email#1739] [Q&A]
Hi noticed react-email doesn't have built-in components for lists and list items. 

I came across this article which suggests it's not trivial for email: https://www.litmus.com/blog/the-ultimate-guide-to-bulleted-lists-in-html-email

I also noticed that there used to be a list component in react-email, but not anymore. What's your advice on lists on emails?

Thanks,

**Comment by gabrielmfern** [comment]
Looked into it, seems trivial to me, honestly. Most of the changes they recommend making is mostly opinionated on where the symbol should be, and the info on the `type` attribute is just normal HTML.

Any other reason we should make these?

**Reply by YousefED** [reply]
Good point. Technically, I think they also explicitly reset the styles, so it makes sense react-email does that for me?

But technical details aside, as a consumer, I think I want to use "react-email powered compoments" to make sure my email renders correctly across different clients - and don't want to figure out the technical details (is element xyz supported across email clients or not? what are the caveats, etc), because that's why I use a library like react-email. TLDR; as a "dumb" consumer I'd benefit from either (a) built-in components representing lists, that I can reuse, or (b) some guidance in documentation that explains which "regular" HTML components are safe to use.

Hope this perspective helps and curious how you think about this!
